src/test/java/net/smartcosmos/usermanagement/role/persistence/RolePersistenceServiceTest.java

```diff
@@ -129,6 +129,58 @@ public void thatUpdateRoleSucceeds() {
     }
 
     @Test
+    public void thatUpdateRoleSucceedsWithSameName() {
+
+        final String roleName = "updateTestRole2";
```

Comment:
I agree, but it's a huge thing to do. You actually need to rip everything that's already there apart and write a lot of new test classes. I didn't want to get down this rabbit hole this time, and therefore added my tests to what is already there in the way the other tests are written.

However, there already is a ticket for test improvements in this service.